Tests/BrowserServicesKitTests/ContentBlocker/AdClickAttribution/AdClickAttributionCounterTests.swift

```diff
@@ -23,7 +23,7 @@ import Persistence
 
 class MockKeyValueStore: KeyValueStoring {
     
-    var store = Dictionary<String, Any?>()
+    var store = [String: Any?]()
```

Comment:
Wow, not sure I've ever seen anyone declare a Dictionary by actually typing dictionary before